apps/deploy_monitoring/src/image.rs

```diff
@@ -76,3 +76,9 @@ pub struct Sandbox;
 impl DeployMonitoringContainer for Sandbox {
     const NAME: &'static str = "deploy-monitoring-tezedge-sandbox-launcher";
 }
+
+pub struct TezedgeMemprof;
```

Comment:
@vlad9486 @adonagy @jurajselep 
do we want this just for TezEdge node or also for Ocaml node?

Reply:
@bkontur Now it work only for tezedge node. The name 'light-node' hardcoded into bpf module.